fw/config.h

```diff
@@ -29,13 +29,29 @@
 #define RING_LIGHT_LEDS 24
 #define LIGHT_BAR_LEDS 16
 
+// tentative names for LED ring modes
+typedef enum {
+  RING_LIGHT_MODE_PLACEHOLDER1, // single colour
+  SPIN,
+  RING_LIGHT_MODE_PLACEHOLDER2, // colour shift
+  RING_LIGHT_MODE_PLACEHOLDER3, // static rainbow
+  RING_LIGHT_MODE_PLACEHOLDER4, // reactive rainbow
+  REACT_TO_SCR,
+  RING_LIGHT_MODE_PLACEHOLDER5, // breathing
+  HID,
+  RING_LIGHT_MODE_ENUMS
```

Comment:
Could you clarify what this does? Is this here to keep track of the number of possible values this enum can take on?

Reply:
Yeah it's just a way to automatically track the number of enums. I'll rename it so that it's clearer what it is.

Reply:
Renamed.